playwright.config.ts

```diff
@@ -0,0 +1,47 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './e2e',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://127.0.0.1:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+    /*
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
```

Comment:
This will probably be needed when I get back https://github.com/open-sauced/ai/pull/258

Reply:
Ok, I uncommented it

Reply:
Actually it still passed with firefox but it's [flaky](https://playwright.dev/docs/test-retries#:~:text=Playwright%20Test%20will%20categorize%20tests,run%20and%20failed%20all%20retries.)
<img width="1185" alt="image" src="https://github.com/open-sauced/ai/assets/47471007/1153aedf-f0d2-4809-9d62-2b28c717a490">

Reply:
We can leave it commented out for now.